api/swagger.yaml

```diff
@@ -1025,7 +1025,7 @@ definitions:
           SecurityOpt:
             type: "array"
             description: "A list of string values to customize labels for MLS
-            systems, such as SELinux."
+              systems, such as SELinux."
```

Comment:
Hm; perhaps change to the same format as we use elsewhere for longer descriptions;

```yaml
            description: |
              A list of string values to customize labels for MLS systems, such
              as SELinux.
```